src/install/lockfile/bun.lock.zig

```diff
@@ -2147,6 +2148,36 @@ fn parseAppendDependencies(
             }
         }
     }
+
+    if (is_root) {
+        workspaces: for (lockfile.workspace_paths.values()) |workspace_path| {
+            for (workspaces_obj.data.e_object.properties.slice()) |prop| {
+                const key = prop.key.?;
+                const value = prop.value.?;
+                const path = key.asString(allocator).?;
+                if (!strings.eqlLong(path, workspace_path.slice(buf.bytes.items), true)) continue;
+
+                const name = value.get("name").?.asString(allocator).?;
+                const name_hash = String.Builder.stringHash(name);
+
+                const dep: Dependency = .{
+                    .name = try buf.appendWithHash(name, name_hash),
+                    .name_hash = name_hash,
+                    .behavior = .{ .workspace = true },
+                    .version = .{
+                        .tag = .workspace,
+                        .value = .{
+                            .workspace = try buf.append(path),
+                        },
+                    },
+                };
+
+                try lockfile.buffers.dependencies.append(allocator, dep);
```

Comment:
```suggestion
                // We don't need to append to resolutions here because ...
                try lockfile.buffers.dependencies.append(allocator, dep);
```